packages/react-magma-dom/src/components/NativeSelect/NativeSelect.tsx

```diff
@@ -22,14 +23,37 @@ export interface NativeSelectProps
   /**
    * @internal
    */
-  testId?: string;
   optionLabel?: string;
+  testId?: string;
 }
-const StyledNativeSelect = styled.select<{
+const StyledNativeSelectWrapper = styled.div<{
+  disabled?: boolean;
+  hasError?: boolean;
+  isInverse?: boolean;
   theme: ThemeInterface;
+}>`
+  ${inputWrapperStyles}
+  padding: 0 ${props => props.theme.spaceScale.spacing03} 0 0;
+  svg {
+    color: ${props =>
+      props.isInverse && props.disabled
+        ? transparentize(0.6, props.theme.colors.neutral100)
+        : props.disabled
+        ? transparentize(0.4, props.theme.colors.neutral500)
+        : 'inherit'};
+    margin: 0 0 0 -${props => props.theme.spaceScale.spacing06};
+    pointer-events: none;
+    z-index: 1;
+  }
+`;
+
+const StyledNativeSelect = styled.select<{
   isInverse?: boolean;
+  theme: ThemeInterface;
 }>`
   ${inputBaseStyles};
+  height: 38px;
+  // Required for Windows && Chrome support
   background: inherit;
```

Comment:
Is this still affecting the border styling?

Reply:
Moved border to <select> to help with spacing.